panther_manager/README.md

```diff
@@ -0,0 +1,72 @@
+```
+
+#### Subscribes
+
+- `/panther/battery` [*sensor_msgs/BatteryState*]: state of internal battery.
+- `/panther/driver/motor_controllers_state` [*panther_msgsDriverState*]: state of motor controllers.
+- `/panther/hardware/e_stop` [*std_msgs/Bool*]: state of emergency stop.
+- `/panther/hardware/io_state` [*panther_msgs/IOState*]: state of IO pins.
+- `/panther/system_status` [*panther_msgs/SystemStatus*]: state of system including CPU temperature and load.
+
+#### Services
+
+- `/panther/manager/overwrite_fan_control` [*std_srvs/SetBool*]: overwrites fan control, setting it to always on.
+
+#### Service clients
+
+- `hardware/aux_power_enable` [*std_srvs/SetBool*]: enables aux power output.
+- `hardware/e_stop_trigger` [*std_srvs/Trigger*]: triggers e-stop.
+- `hardware/fan_enable` [*std_srvs/SetBool*]: enables fan.
+
+#### Parameters
+
+- `~battery_window_len` [*int*, default: **6**]: moving average window length used to smooth out temperature readings of battery.
+- `~cpu_fan_off_temp` [*float*, default: **60.0**]: temperature in **deg C** of CPU, above which the fan is turned on.
+- `~cpu_fan_on_temp` [*float*, default: **70.0**]: temperature in **deg C** of any drivers above which the fan is turned on.
+- `~cpu_window_len` [*int*, default: **6**]: moving average window length used to smooth out temperature readings of CPU.
+- `~critical_bat_temp` [*float*, default: **59.0**]: temperature of battery above which robot enters e-stop state.
+- `~default_identity_file` [*string*, default: **~/.ssh/id_rsa**]: 
+ default path used to find identity global file for SSH. 
+- `~driver_fan_off_temp` [*float*, default: **35.0**]: temperature in **deg C** of any drivers below which the fan is turned off.
+- `~driver_fan_on_temp` [*float*, default: **45.0**]: temperature in **deg C** of any drivers above which the fan is turned on.
+- `~driver_window_len` [*int*, default: **6**]: moving average window length used to smooth out temperature readings of each driver.
+- `~fatal_bat_temp` [*float*, default: **62.0**]: temperature of battery above which robot shuts down.
+- `~high_bat_temp` [*float*, default: **55.0**]: temperature of battery above which robots starts displaying warning log.
+- `~hosts` [*list*, default: **empty list**]: list of hosts to request shutdown.
+  - `cmd` [*string*, default: **sudo shutdown now**]: command executed to on shutdown of given device.
+  - `identity_file` [*string*, default: **None**]: SSH identity file global path. If not set, defaults to `~default_identity_file`.
+  - `ip` [*string*, default: **None**]: IP of a host to shutdown over SSH.
+  - `username` [*string*, default: **None**]: username used to log in to over SSH.
+- `~hysteresis` [*float*, default: **60.0**]: minimum time of fan being turned on.
+- `~overwrite_fan_control` [*bool*, default: **False**]: enable fan to be always on start of the node. It can be turned off later by `/panther/manager/overwrite_fan_control`.
```

Comment:
```suggestion
- `~overwrite_fan_control` [*bool*, default: **False**]: enable the fan to be always on at start of the node. It can be turned off later by `/panther/manager/overwrite_fan_control` service.
```